app/javascript/components/Login/Login.jsx

```diff
@@ -75,7 +83,7 @@ class Login extends React.Component {
         data.append('user[password]', password);
 
         Requests.post('/users', data).then(() => {
-            this.login()
+            this.login(null, username, password)

```

Comment:
because you are calling this.login() directly, you need to pass it the username and password data.